src/OSPSuite.SimModel/SimModelImportDefinitions.cs

```diff
@@ -5,11 +5,11 @@ namespace OSPSuite.SimModel
 {
    internal class SimModelImportDefinitions
    {
-
-#if _WINDOWS
+#if NETFRAMEWORK
       public const String NATIVE_DLL = "OSPSuite.SimModelNative.dll";
+
 #else
-      public const String NATIVE_DLL = "libOSPSuite.SimModelNative.so";
+      public const String NATIVE_DLL = "OSPSuite.SimModelNative";
```

Comment:
in case of netframework we have to specify the file name, this is Windows only now so we don't need .so version.

Under Ubuntu, we can count on a newer dot net runtime, so we can use the convention.